### ISSUE TITLE
Feature suggestion: right-click menu for string constants, to set state

### ISSUE THREAD

**Issue by kosist** [JKISoftware/JKI-State-Machine-Editor#103]
In case, when one needs to set transition to some existing state, he needs to specify that state in string constant. Existing state could be copied, then new string constant created, and then copied text is pasted into string constant (or typed manually).

But, maybe it could be usefult to have the following: when user right-clicks on string constant (maybe it is enough if it will be empty one, not on constants with text), there will be additional menu like "Add existing state" -> which will have submenu with list of existing states (frames, macros). On select, string will be pasted into string constant.

Also, just to avoid having this submenus on string constants, which are not related to state machine, it could be checked if it is connected by wire nodes to "Add State(s) to Queue__JKI_lib_State_Machine" function.

Thank you very much,

Sincerely, Ivan.

**Comment by jimkring** [comment]
Yes. It certainly would be nice for developers to be able to insert states
via right-clicking on the string constant -- I would use that all the time.

Thanks for sharing your great ideas!
-- 
Jim Sent from my mobile.
